samwise.js

```diff
@@ -3,12 +3,12 @@
  *
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.